eslint.config.js

```diff
@@ -62,6 +62,7 @@ export default ts.config(
         }
       ],
       'react/prop-types': 'off',
+      'quotes': ['error', 'single', { 'avoidEscape': true, 'allowTemplateLiterals': true }],
```

Comment:
This is a rule that we had before which makes sense to restore, otherwise we get mixed quotes in imports.